daffodil-japi/src/main/scala/org/apache/daffodil/japi/Daffodil.scala

```diff
@@ -98,40 +100,40 @@ object Daffodil {
 
 }
 
-/**
- * Compile DFDL schemas into [[ProcessorFactory]]'s or reload saved parsers into [[DataProcessor]]'s.
- *
- * Do not use the Compiler constructor to create a Compiler. Instead, use [[Daffodil#compiler()]].
- */
+///**
```

Comment:
Temporary - had to comment out all the javadoc here. It was preventing the japi jar from being created. I have no idea why it was failing when it worked before.